gatsby-node.js

```diff
@@ -0,0 +1,8 @@
+exports.onCreateWebpackConfig = ({actions, getConfig}) => {
+    // Hack due to Tailwind ^1.1.0 using `reduce-css-calc` which assumes node
```

Comment:
@MaximumCrash Q: If we're not using Tailwind - do we need this?

Reply:
Nah we don't need this. I'll remove it for ya.